Gruntfile.js

```diff
@@ -2,7 +2,20 @@ module.exports = function( grunt ) {
 
 	'use strict';
 	var remapify = require('remapify');
-	var banner = '/**\n * <%= pkg.homepage %>\n * Copyright (c) <%= grunt.template.today("yyyy") %>\n * This file is generated automatically. Do not edit.\n */\n';
+	var banner   = '/**\n * <%= pkg.homepage %>\n * Copyright (c) <%= grunt.template.today("yyyy") %>\n * This file is generated automatically. Do not edit.\n */\n';
+
+	// Path to WordPress install. Either absoloute or relative to this plugin.
+	// Change this by passing --abspath="new/path" as a grunt option.
+	var abspath;
+
+	if ( grunt.option( "abspath" ) ) {
+		abspath = grunt.option( "abspath" );
+	} else if ( 'WP_DEVELOP_DIR' in process.env ) {
+		abspath = process.env.WP_DEVELOP_DIR;
+	} else {
+		abspath = '/tmp/wordpress';
```

Comment:
Can we perform a check to ensure `abspath` actually exists, and show a helpful error if it doesn't?